### PR TITLE
Enum can have boolean values

### PR DESCRIPTION
**Pull request by momala454**

### CODE REVIEW

**Comment by DerManoMann** [comment]
You will need to update the annotation classes too to make the tests pass...

**Comment by momala454** [comment]
thanks done

File: src/Annotations/ServerVariable.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ class ServerVariable extends AbstractAnnotation
     /**
      * An enumeration of values to be used if the substitution options are from a limited set.
      *
-     * @var string[]|int[]|float[]|\UnitEnum[]|class-string
+     * @var string[]|int[]|float[]|\UnitEnum[]|class-string|bool[]
```
Comment (DerManoMann) [inline src/Annotations/ServerVariable.php:30]: Ah, just a minor thing, but one I'd like to see changed for consistency - could you move the `bool[]` adjunct to `string`,`int`,`float`, please, so the native types are all together?
 ```suggestion
     * @var string[]|int[]|float[]|bool[]|\UnitEnum[]|class-string
```

**Review by DerManoMann — CHANGES_REQUESTED** [review]
One more find/replace that it looks good to go.

**Review by DerManoMann — APPROVED** [review]

**Comment by DerManoMann** [comment]
Thanks @momala454

**Pull request merged**